cluster-autoscaler/cloudprovider/clusterapi/README.md

```diff
@@ -417,8 +417,6 @@ spec:
        ## replicas: 1
 ```
 
-**Warning**: If the Autoscaler is enabled **and** the replicas field is set for a `MachineDeployment` or `MachineSet` the Cluster may enter a broken state where replicas become unpredictable.
```

Comment:
We enabling the autoscaler and setting the replica fields mutually exclusive last year: https://github.com/kubernetes-sigs/cluster-api/pull/10370

So I don't think we have to warn about something that our webhook is blocking